backend/src/main/resources/application.properties

```
@@ -1,6 +1,6 @@
 spring.application.name=calc
 
-spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.url=jdbc:h2:file:./src/main/resources/db/calculator_db;DB_CLOSE_DELAY=-1
```

nit about DB file location. `src/main/kotlin/.../resources` directory contains static non-executable data, which is contained in the classpath of the application. In our case it's `application.properties` but can also be bundled JSON data, etc. I think this is not the best place for dynamic DB file.

I think we can go as simple as

```suggestion
spring.datasource.url=jdbc:h2:file:./db/calculator_db;DB_CLOSE_DELAY=-1
```

Note that if and when we will be deploying our application (#20), we can always override Spring configuration to set more suitable DB location.